### ISSUE TITLE
mic_in

### ISSUE THREAD

**Issue by balthisar** [edent/Sercomm-API#3]
Any idea how to enable mic_in?

I've tried both the AUDIO and USER groups obvious settings, but I can't seem to enable the mic. My OC810-ADT does have a microphone and a speaker.

By the way, _great_ resource here!

**Comment by edent** [comment]
Glad this is of help.

Are you able to hear any audio when you stream video?

**Comment by balthisar** [comment]
No, but I see that the microphone appears to be disabled. When I enable the UI, there's no microphone option. When I do

`util/query.cgi`

I get (amongst the other results)

~~~
mic_in=off
speaker_out=off
~~~

There's also:

`adm/get_group.cgi?group=AUDIO`, yielding:

~~~
[AUDIO]
audio_in=0
in_volume=1
in_audio_type=1
audio_out=0
out_volume=1
out_audio_type=0
audio_mode=1
operation_mode=1
~~~

I can try `adm/set_group.cgi?group=AUDIO&audio_in=1`, but then requerying still results in the mic being disabled in software.
